src/components/interaction-designer/block-editors/ContactPropertyEditor.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="contact-property-editor">
+    <div class="form-group">
+      <label>{{ 'flow-builder.action-label' | trans }}</label>
+      <p>{{ 'flow-builder.contact-property-action-hint' | trans }}</p>
+      <div class="form-group">
+        <div class="custom-control custom-radio">
+          <input
+            id="setProp"
+            v-model="propertyAction"
+            type="radio"
+            name="contactPropAction"
+            :value="PROPERTY_ACTION.SET"
+            class="custom-control-input">
+          <label
+            class="custom-control-label font-weight-normal"
+            for="setProp">
+            {{ 'flow-builder.set-contact-property' | trans }}
+          </label>
+        </div>
+        <div class="custom-control custom-radio">
+          <input
+            id="clearProp"
+            v-model="propertyAction"
+            type="radio"
+            name="contactPropAction"
+            :value="PROPERTY_ACTION.CLEAR"
+            class="custom-control-input">
+          <label
+            class="custom-control-label font-weight-normal"
+            for="clearProp">
+            {{ 'flow-builder.clear-contact-property' | trans }}
+          </label>
+        </div>
+      </div>
+
+      <validation-message
+        #input-control="{ isValid }"
+        :message-key="`block/${block.uuid}/config/set_contact_property/property_key`">
+        <div class="block-contact-property-key">
+          <text-editor
```

Comment:
How come we no longer allow selection from builder.config.json properties? This may break the import validations - this should be tested - as we check there that block properties exist in the org (e.g. are loaded in the builder.config.json)

Reply:
@jcbashdown , Jon & Amen confirmed that last week to change the dropdown selector into an open input field. If I'm not wrong Jon had a discussion with Mark about it.
To my understanding, we want this change for community version which should have allow users to enter any `contact prop` (the dropdown doesn't allow user to add contact prop for now). But in the future, we would probably go back to the dropdown. Need to confirm that with Jon though, because Brett asked the same question during retro meeting